content/windows.md

```diff
@@ -430,43 +221,84 @@ The next page will ask what type of installation you want to perform. Since ther
 
 ![Windows installation types](/images/dual-booting/windows-installation-type.jpg)
 
-Finally, the installer will ask you to select where you want to install to. In this scenario, we are installing Windows to a pre-made partition on the shared drive (if you're doing something else, see [Planning the Installation](#planning-the-installation)).
+Finally, the installer will ask you to select where you want to install to. Please follow the instructions for the configuration you chose under [Dual-Booting Windows](#dual-booting-windows-with-linux)
```

Comment:
It's a pet peeve, but we don't want to end a sentence with a preposition.

"Finally, the installer will ask you to select the location where you want Windows installed."